biz.aQute.repository/src/aQute/bnd/deployer/repository/wrapper/Plugin.java

```diff
@@ -153,7 +153,11 @@ public void close() throws java.io.IOException {
 	}
 
 	public String toString() {
-		return wrapper != null ? wrapper.toString() : "<wrapper not set>";
+		if (wrapper != null) {
+			wrapper.init();
```

Comment:
This is wrong. Calling toString should not init a repo. toString is called in debuggers and it should not have the side effect like this.

Also, the wrapper is basically deprecated. As is jpm repo.